rpc/src/v1/types/transaction.rs

```diff
@@ -67,6 +69,10 @@ pub struct Transaction {
     pub s: U256,
     /// Transaction activates at specified block.
     pub condition: Option<TransactionCondition>,
+    /// transaction type
+    pub tx_type: u8,
+    /// optional access list
+    pub optional_access_list: Option<Vec<(H160, Vec<H256>)>>,
```

Comment:
```suggestion
    pub access_list: Vec<(H160, Vec<H256>)>,
```

I do not see using `optional_` prefix in other parts of the code for vec, and in EIP2929 the list is named `access_list`. So I propose to name it `access_list`.

Since `Vec::new()` does not allocate until any element is inserted  (https://doc.rust-lang.org/beta/std/vec/struct.Vec.html#method.new), makes sense to be an `Option`?

Reply:
Agreed